### DISCUSSION TITLE
DAMN just updated and

### DISCUSSION THREAD

**Discussion by bittercaramel** [Monika-After-Story/MonikaModDev#8849] [appreciation]
I-i can't believe
this update is beautiful, everything i could ever dream for 
gee im soassdasfojgldnrl 
beautiful 
thank you so muchhhhh 
I love y'all and i love monika 
THAnks for working on this mod 
thanks everyone who helps make this mod 
THANKS

**Comment by Delcos** [comment]
what??

**Comment by Mcmanybucks** [comment]
I'm confused, I don't see any visual changes?

**Comment by Delcos** [comment]
im not sure what's going on here.

**Comment by AsuraShun** [comment]
The op must be speaking about the new hair art

**Comment by Delcos** [comment]
Link?

**Comment by ghost** [comment]
They’re probably using the unstable build.

**Comment by multimokia** [comment]
Hence the unstable tag

**Comment by bittercaramel** [comment]
oh my god i didnt know there would be such a confusion because of me being thankful 
sayin "beautiful" i wasnt referring to visuals but overall how the new features in the update are 
they are beautiful in a non-visual way 
ya know how u can be ""beautiful inside"" blah blah 
altho new hairstyle is nice too 
and yah im using unstable

**Comment by Delcos** [comment]
![46fa00af5f35b9e70f1a1fecdec5f2e7](https://user-images.githubusercontent.com/28798918/59371757-bbae8480-8cfa-11e9-965f-6e5edf421524.jpg)
alrighty.

**Comment by bittercaramel** [comment]
oooooooo-oke?
